### PR TITLE
Hostname verification support on ssl configuration

### PR DESCRIPTION
**Pull request by acdemyers**

### CODE REVIEW

**Comment by acdemyers** [comment]
#build

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -47,6 +47,8 @@ repertiore.HIGH=Cipher suites 3DES and 128 bit and higher
 repertiore.MEDIUM=Cipher suites DES and 40 bit
 repertiore.LOW=Cipher suites without encryption
 repertiore.CUSTOM=Custom list of cipher suites
+repertoire.verifyHostname=Perform host name verification for outbound SSL connections using a specific SSL configuration
+repertoire.verifyHostname.desc=Specifies whether host name verification is enabled for outbound connection on using a specific SSL configuration is enabled.  If set to true, all outbound SSL connection using the specified SSL configuration will undergo verification of the target server host name against that server's certificate.  The attribute is false by default.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: Change 
"Specifies whether host name verification is enabled for outbound connection on using a specific SSL configuration is enabled.  If set to true, all outbound SSL connection using the specified SSL configuration will undergo verification of the target server host name against that server's certificate.  The attribute is false by default."
to
"The verifyHostname attribute specifies whether host name verification for outbound connections using a specific SSL configuration is enabled. If set to true, then all outbound SSL connections that use the specified SSL configuration undergo verification of the target server host name against that server's certificate. The attribute is set to false by default."

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -47,6 +47,8 @@ repertiore.HIGH=Cipher suites 3DES and 128 bit and higher
 repertiore.MEDIUM=Cipher suites DES and 40 bit
 repertiore.LOW=Cipher suites without encryption
 repertiore.CUSTOM=Custom list of cipher suites
+repertoire.verifyHostname=Perform host name verification for outbound SSL connections using a specific SSL configuration
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: Change
"Perform host name verification for outbound SSL connections using a specific SSL configuration"
to
"The verifyHostname attribute performs host name verification for outbound SSL connections using a specific SSL configuration."

**Comment by andrea-pichler** [comment]
No message review from L2 required.

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 2 above)
Comment (acdemyers) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: @Charlotte-Holt I've update the metetype.properties file with your suggested changes.  Thank you for you comments.

**Comment by acdemyers** [comment]
#build

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 2 above)
Comment (Charlotte-Holt) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: @acdemyers Label added. Thank you!

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (acdemyers) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: @Charlotte-Holt I've update the metetype.properties file with your suggested changes. Thank you for you comments.

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Pull request merged**